src/basic.js

```diff
@@ -271,6 +289,14 @@ function ForceHLAllObjects() {
 	HLAllObjects();
 }
 
+/**
+ * Delay then Force Highlight objects
+ *  This is used to wait for DOM to populate before filling in problem details in left panel.
+ */
+function delayForceHLAllObjects() {
+	setTimeout(ForceHLAllObjects, 100);
```

Comment:
The delay looks fragile, maybe we should wait until the object appears? A quick search: https://stackoverflow.com/questions/5525071/how-to-wait-until-an-element-exists

Reply:
yes, it is a little fragile. But this is being called on selectionchanged, so we dont know for certain what element we might need.